bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3995,6 +4102,7 @@ ReplicaImp::ReplicaImp(bool firstTime,
       viewChangeProtocolEnabled{config.viewChangeProtocolEnabled},
       autoPrimaryRotationEnabled{config.autoPrimaryRotationEnabled},
       restarted_{!firstTime},
+      MAIN_THREAD_ID{std::this_thread::get_id()},
```

Comment:
Where do you use it?

Reply:
Currently not using it but it will on the next steps of this feature.